### ISSUE TITLE
Missing predeploy script causing deployment to fail on 0.14.0

### ISSUE THREAD

**Issue by eliias** [dokku/dokku#3385]
## Description of problem

I have upgraded to 0.14.0 and got a deployment issue for a project that otherwise was working flawlessly. After enabling trace I was still not able to find out what's going until I saw this one. The predeploy command looks like this `rm -rf ''`.

```bash
remote: + [[ predeploy == \r\e\l\e\a\s\e ]]
remote: ++ get_phase_script dokku/api:latest predeploy
remote: + SCRIPT_CMD=
remote: + rm -rf ''
remote: + exit_code=1
remote: + set -e
remote: + [[ 1 -eq 10 ]]
remote: + implemented=1
remote: + [[ 1 -ne 0 ]]
remote: + exit 1
```

### How reproducible

This is a Dockerfile deployment but I have also tried with a buildpack deployment to confirm the general occurence of this issue.

If someone has a deploy script in `app.json` (e.g. I use "postdeploy" for migrations) it will fail as long as `predeploy` is not defined. If I add `echo $PWD` or something else, it will build and deploy as expected. The only thing I am not 100% sure about, I can't remember if this project ever had a `predeploy` and if this information got cached somewhere? I have cleaned the build caches on the server and even removed all docker intermediate images to rule out caching but tbh, this is a bit outside of my territory.

I also removed the bare repo on the server and reinitialized it, cause a corrupt git repo was my initial thought.

### Steps to Reproduce

1. add `app.json`
2. add only a "postdeploy" script
3. next push should fail

#### Actual Results

```
Successfully built 498920263817
Successfully tagged dokku/api:latest
-----> Releasing api (dokku/api:latest)...
-----> Deploying api (dokku/api:latest)...
To xxx:api
 ! [remote rejected] master -> master (pre-receive hook declined)
error: failed to push some refs to 'dokku@xxx'
```

#### Expected Results

```bash
-----> Shutting down old containers in 60 seconds
=====> b3cd96c1e08aedd1954a452ba77db09a968edce59b2af27518a7c4b3d2a49213
=====> 8d5403aa5a5802bc722ed733a730643037c8e12fbf31d9e912b692c88f6085d7
=====> Application deployed:
       http://api.com

To xxx:api
 + bd1a6eb...c9175b9 master -> master (forced update)
```

## Environment Information

- Ubuntu 18.04
- dokku 0.14.0

### `dokku report APP_NAME` output

Please let me know if you really need this. I don't think it contains any relevant information regarding this issue.

### How (deb/make/rpm) and where (AWS, VirtualBox, physical, etc.) was Dokku installed?:

Fresh Ubuntu 18.04 on a bare metal server.

### Additional information

- If a deploy is failing or behaving unexpectedly:
  - The type of application being deployed: Ruby on Rails buildpack and custom Dockerfile

Please let me know if I can be of any help. If you point me in the right direction on how I can track down `app.json` commands, I'll do so.

**Comment by josegonzalez** [comment]
Closing as there is a pul request open. I'll get the fix released as soon as the new tests pass!

**Comment by josegonzalez** [comment]
Fixed in 0.14.1.
